packages/shared/src/components/MainLayout.tsx

```diff
@@ -89,6 +91,12 @@ export default function MainLayout({
           />
         )}
         <div className="flex-1" />
+        {mobileTitle && (
+          <strong className="block laptop:hidden typo-callout">
```

Comment:
Shouldn't this be `hx` element?

Reply:
I'm not sure what that means?

Reply:
h1, h2, h3, h something

Reply:
We generally don't set the callout to a heading from what I can see.

Happy to change to a H3 as this is what it is on desktop, WDYT?
Do keep in mind this section will also render the good morning message (which is a span)
And the rank, (also a span)

Reply:
H3 sounds good